src/ontology/cl-edit.owl

```diff
@@ -28247,6 +28248,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
+# Class: obo:CL_4032000 (club-like cell of the urethra epithelium)
```

Comment:
Why is the label 'club-like cell of ....' and not ' club cell of ...'?
I skimmed the 3 dbxref annotations on the definition and did not see the "like" terminology used.

Reply:
club cell has implications of it being lung cell type. While it is used generally as such, I think for the ontology (which doesn't have context like in the papers) I wanted to be careful not to be misleading. The cell type is only called club cell cause of its similarity to lung type, hence the use of "-like". Totally why people hate ontology names lol

Reply:
did that heaps for BDSO when names were misleading :) also see http://purl.obolibrary.org/obo/CL_4029003 happy to add to the comment the choice of "-like"

Reply:
I see it is also not a subclass of 'club cell', which if it were, would be problematic because of the way club cell is defined. Down the road, perhaps 'club cell' will be a new superclass and the current 'club cell' will become 'club cell of the bronchiole'....

Reply:
> http://purl.obolibrary.org/obo/CL_4029003

FYI, after clicking the link: "Term Not Found" 
But I do see it in owl-edit.owl

Reply:
lol yeah, i started by putting it as a subclass of club cell then realise it was super weird and hence this whole thing changed to a way more complicated ticket >.< was hoping this would be an easy one for @ar-ibrahim to handle, but of course nothing ends up straightforward lol 
I think I'd rather not club cell as a new superclass, as usage-wise, it seems that "club cells" is more widely used to mean "club cell of the bronchiole". I guess its the messy art of finding an in between of being accurate yet trying to represent the messy world of biology. This is totally why whole workshops have been had on naming cell types.

Reply:
> FYI, after clicking the link: "Term Not Found"

that.... is not good :( sigh, ill investigate and figure out if its our side or ontobee side

Reply:
oh lol, its a more recently added term, not in release yet, hence not in ontobee, no worries

Reply:
"**Prostate club cells** are similar to lung club cells in their enrichment of immunomodulatory programs" PMID:[30566875](https://pubmed.ncbi.nlm.nih.gov/30566875)

'prostate club cell' seems like a valid synonym (or a candidate for rdfs:label).

Reply:
I have the same issue on that with club cell of urethral epithelium, it still says club cell when its not REALLY a club cell (or at least how we use it), but will add as synonym

Reply:
> its not REALLY a club cell (or at least how we use it)

That is what my comment is probing -- the way CL defines 'club cell' (i.e., only restricted to lung).